### ISSUE TITLE
Jacobian support

### ISSUE THREAD

**Issue by zxhuang97** [NVlabs/curobo#328]
Hi, thank you so much for creating this wonderful library. 

I noticed that `calculate_jacobian` is currently disabled for the forward kinematics. I'm wondering if there's a timeline when this will be available. Thanks!

**Comment by balakumar-s** [comment]
Yes, it's not easy yet. We did think of adding it but haven't figure out how best to implement it. What use case do you have in mind for using Jacobian?
